packages/app/src/connecting/components/connecting.component.tsx

```diff
@@ -60,7 +60,13 @@ const Connecting: FunctionComponent<{
 
   useEffect(() => {
     const timeout = setTimeout(() => {
-      if (unlocked && loaded && syncInitialized) {
+      if (
+        unlocked &&
+        loaded &&
+        syncInitialized &&
+        (syncState === SynchronizationState.Cache ||
+          SynchronizationState.Loaded)
+      ) {
```

Comment:
according to the reducer logic, the `syncInitialized` can be true only when `syncState` is set to `Cache` or `Loaded`. This logic no handle the Harmony 😉

Reply:
Fixed in [6e9f68e](https://github.com/mudita/mudita-center/pull/642/commits/6e9f68e9693fe7bc9971fdb61d6aa68c65fe95c2)